lnrpc/neutrinorpc/neutrino_server.go

```diff
@@ -423,3 +427,21 @@ func (s *Server) getBlock(hash chainhash.Hash) (*GetBlockResponse, error) {
 		RawHex:            blockData,
 	}, nil
 }
+
+// GetBlockHash returns the header hash of a block at a given height.
+//
+// NOTE: Part of the NeutrinoKitServer interface.
+func (s *Server) GetBlockHash(ctx context.Context,
+	in *GetBlockHashRequest) (*GetBlockHashResponse, error) {
+
+	if s.cfg.NeutrinoCS == nil {
+		return &GetBlockHashResponse{}, ErrNeutrinoNotActive
```

Comment:
nit: in case of an error, the first return value should be nil in RPC responses.

Reply:
Done. Changed it for all other `neutrinorpc` functions as well.